proto/p4/config/v1/p4info.proto

```diff
@@ -106,6 +106,8 @@ message P4Ids {
 
     // externs for other architectures (vendor extensions)
     OTHER_EXTERNS_START = 0x80;
+    MATCH_LOOKUP_TABLE = 0x81;
```

Comment:
This will conflict with existing vendors having their externs starting with 0x81. Is this going to be PNA specific or being added to P4 language core ?
If PNA, then we should probably think along the lines of having a pna.p4 to extend this so that other possible vendor architectures can extend from 0x81 onwards.
If core, then why not 0x18 and 0x19 directly ?